knexfile.js

```diff
@@ -0,0 +1,35 @@
+// Update with your config settings.
+
+const pgConfig = {
+  client: 'pg',
+  timezone: 'UTC',
+  connection: {
+    host: process.env.DB_HOST || 'localhost',
```

Comment:
Rather than guessing decided to comment. Wonder why we are not using `env.get('DB_PORT')` as in `.ts` files

Reply:
Because this is only looked at when running the migrations. When we do that we only want the db ens not the whole set of envs the applications needs. I could write a separate envalid loader for that but it's not really needed